spec/jobs/export_user_archive_spec.rb

```diff
@@ -331,7 +331,7 @@ def make_component_json
           .category_users
           .where(category_id: category_id)
           .first_or_initialize
-          .update!(last_seen_at: reset_at)
+          .update!(last_seen_at: reset_at, notification_level: NotificationLevels.all[:regular])
```

Comment:
weird cause I am not sure what regular means, maybe this test needs some rewriting?